internal/operators/lvm/config.go

```diff
@@ -7,9 +7,9 @@ import (
 const (
 	// LvmMinOpenshiftVersion is the minimum OCP version in which lvmo is supported
 	// Any changes here should be updated at line 16 too.
-	LvmoMinOpenshiftVersion                           string = "4.11.0"
-	LvmsMinOpenshiftVersion                           string = "4.12.0"
-	LvmsMinOpenshiftVersionForNewResourceRequirements string = "4.13.0"
+	LvmsMinOpenshiftVersion                            string = "4.11.0"
```

Comment:
Why do we have 3 different `minVersions`?

Reply:
so there is no difference between `lvms` and `lvmo` they are both called `lvm`.

the only place that has a differtents is in the OLM where they are two different bundles 
so they only place you will see it is in the [manifest](https://github.com/openshift/assisted-service/blob/master/internal/operators/lvm/manifest.go#L38-L55)

Reply:
Not suggesting to remove, just to name accordingly and clearly.

Why do we have a constant if we have a config set from env var with default values?
What's the difference between constant `LvmMinOpenshiftVersion` and env var `LVM_MIN_OPENSHIFT_VERSION`?
I think it's important that we make all these "min versions" as less confusing as possible.

Reply:
there is only one min version

Reply:
```
        LvmsMinOpenshiftVersion                            string = "4.11.0"
	LvmsMinOpenshiftVersion4_12                        string = "4.12.0"
	LvmsMinOpenshiftVersion_ForNewResourceRequirements string = "4.13.0"
```

plus all the config values, which it's not clear what's the relation between constant "min version" and env var "min version"

Reply:
we talked offline, we need to release the version 
I will create a new PR for naming and better usage